vm/internal/handler/export.go

```diff
@@ -39,6 +39,37 @@ func (h *Handler) ExportVolume(ctx echo.Context) error {
 	log.Infof("path: %s", path)
 	log.Infof("fileName: %s", fileName)
 
+	// Check if the target file already exists
+	targetFilePath := filepath.Join(path, fileName)
+	if _, err := os.Stat(targetFilePath); err == nil {
+		// File exists, prompt for confirmation
+		fmt.Println("A file with the same name already exists in the target directory.")
+		fmt.Println("Do you want to:")
+		fmt.Println("1. Overwrite the existing file (O)")
+		fmt.Println("2. Keep both files (K)")
+		fmt.Println("3. Cancel the export (C)")
+
+		var userInput string
+		fmt.Scanln(&userInput)
```

Comment:
**question**: I am not sure to understand how this can work. How the user could do anything at this point? Since this is in a handler it is not interactive, the handler treats an HTTP request and return an HTTP response.